### ISSUE TITLE
Display date on blog posts

### ISSUE THREAD

**Issue by darribas** [urbangrammarai/urbangrammarai.github.io#27]
I just realised we do not display the date of publication on individual blog post pages ([e.g.](https://urbangrammarai.xyz/blog/post23_gisruk.html)). We _do_ display it on the [list of of posts](https://urbangrammarai.xyz/blog/archive). Could we get it also displayed on each post's page just beneath the main title?

**Comment by martinfleis** [comment]
I would love to, though I have currently no clue how. One thing I tried did not work. I would probably need to edit our sphinx theme template to load the metadata during `sphinx-build`. And I am not sure this will get enough priority over other things :). I

**Comment by darribas** [comment]
No, not a big priority if it takes a lot of effort. I was thinking, tweaking the templates for the posts? Perhaps this one?

https://github.com/urbangrammarai/urbangrammarai.github.io/blob/e41d4e117ba4ef8750f910dd4b356b7924e7a10e/src/_templates/bodyheader.html#L8

We could add to the title the date if there is such variable?

**Comment by martinfleis** [comment]
That is not the correct template file and we currently not have one for individual blogposts. Would need to create one to override the existing.
